IBPSA/Fluid/SolarCollectors/BaseClasses/PartialSolarCollector.mo

```diff
@@ -0,0 +1,334 @@
+    annotation(Dialog(group="Configuration declarations", enable= (sysConfig == IBPSA.Fluid.SolarCollectors.Types.SystemConfiguration.Array)));
+
+  Modelica.Blocks.Interfaces.RealInput shaCoe_in if use_shaCoe_in
+    "Shading coefficient"
+    annotation(Placement(transformation(extent={{-140,60},{-100,20}})));
+
+  IBPSA.BoundaryConditions.WeatherData.Bus weaBus "Weather data bus"
+    annotation (Placement(
+    transformation(extent={{-110,70},{-90,90}})));
+  IBPSA.BoundaryConditions.SolarIrradiation.DiffusePerez HDifTilIso(
+    final outSkyCon=true,
+    final outGroCon=true,
+    final til=til,
+    final azi=azi,
+    final rho=rho) "Diffuse solar irradiation on a tilted surface"
+    annotation (Placement(transformation(extent={{-80,70},{-60,90}})));
+
+  IBPSA.BoundaryConditions.SolarIrradiation.DirectTiltedSurface HDirTil(
+    final til=til,
+    final azi=azi) "Direct solar irradiation on a tilted surface"
+    annotation (Placement(transformation(extent={{-80,40},{-60,60}})));
+
+  IBPSA.Fluid.Sensors.MassFlowRate senMasFlo(redeclare package Medium =
+    Medium, allowFlowReversal=allowFlowReversal) "Mass flow rate sensor"
+    annotation (Placement(transformation(extent={{-90,-11},{-70,11}})));
+  IBPSA.Fluid.FixedResistances.PressureDrop res(
+    redeclare final package Medium = Medium,
+    final from_dp=from_dp,
+    final show_T=show_T,
+    final m_flow_nominal=m_flow_nominal,
+    final allowFlowReversal=allowFlowReversal,
+    final linearized=linearizeFlowResistance,
+    final homotopyInitialization=homotopyInitialization,
+    deltaM=deltaM,
+    final dp_nominal=dp_nominal_final) "Flow resistance"
+    annotation (Placement(transformation(extent={{-60,-10},{-40,10}})));
+
+  // The size of the liquid volume has been increased to also add
+  // the heat capacity of the metal.
+  IBPSA.Fluid.MixingVolumes.MixingVolume vol[nSeg](
+    each nPorts=2,
+    redeclare package Medium = Medium,
+    each final m_flow_nominal=m_flow_nominal,
+    each final energyDynamics=energyDynamics,
+    each final p_start=p_start,
+    each final T_start=T_start,
+    each final m_flow_small=m_flow_small,
+    each final V=CTot/cp_default/rho_default*nPanels_internal/nSeg,
+    each final massDynamics=massDynamics,
+    each final X_start=X_start,
+    each final C_start=C_start,
+    each final C_nominal=C_nominal,
+    each final mSenFac=mSenFac,
+    each final allowFlowReversal=allowFlowReversal,
+    each final prescribedHeatFlowRate=false)
+    "Volume of fluid in one segment of the solar collector"
+    annotation (Placement(transformation(
+        extent={{10,-10},{-10,10}},
+        rotation=180,
+        origin={50,-20})));
+
+  Modelica.Thermal.HeatTransfer.Sensors.TemperatureSensor temSen[nSeg]
+    "Temperature sensor"
+    annotation (Placement(transformation(
+      extent={{-10,10},{10,-10}},
+      rotation=180,
+      origin={0,-20})));
+
+  HeatTransfer.Sources.PrescribedHeatFlow QGai[nSeg] "Rate of solar heat gain"
```

Comment:
Used to be `Buildings.HeatTransfer.Sources.PrescribedHeatFlow`, but this isn't a model in the IBPSA library.
This should become `Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow`